nilearn/decoding/space_net.py

```diff
@@ -18,6 +18,7 @@
 from functools import partial
 import numpy as np
 from scipy import stats, ndimage
+from sklearn.exceptions import NotFittedError
```

Comment:
Is it compliant with the minimal version of sklearn requested? I think this error is something recent.